.github/workflows/go-update.yml

```diff
@@ -23,7 +23,10 @@ on:
         type: string
         default: ''
 
-permissions: {}
+permissions:
```

Comment:
If I'm correct that's set just below on the job at lines 37 to 39 for the whole job (contents: write to create branch and pull-requests:write to write into the PR)